src/plugins/InsertTimestamps/modal.tsx

```diff
@@ -12,13 +12,15 @@ const {
     Tooltip,
     Select,
     openModal
-} = BdApi.Webpack.getModule(m => m.ModalContent);
+} = BdApi.Webpack.getModule(m => m.ModalContent && m.Tooltip);
 
 const Parser = BdApi.Webpack.getModule(m => m.parseTopic);
 const PreloadedUserSettings = BdApi.Webpack.getModule(m => m.ProtoClass?.typeName.endsWith("PreloadedUserSettings"), {
     searchExports: true
 });
+const CalendarIcon = BdApi.Webpack.getByKeys("CalendarIcon")?.CalendarIcon;
 const ButtonWrapperClasses = BdApi.Webpack.getModule(m => m.buttonWrapper && m.buttonContent);
+const ButtonClasses = BdApi.Webpack.getModule(m => m.emojiButton && m.stickerButton);
```

Comment:
should use getByKeys here (and above, this code was written before the addition of these methods)

Reply:
Ah, that makes sense. I was a bit worried it was a style choice